play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukBackLink.scala.html

```diff
@@ -18,7 +18,7 @@
 
 @(params: BackLink)(implicit messages: Messages)
 @import params._
-<a href="@{"#".?:(href)}" class="@toClasses("govuk-back-link", classes)"@toAttributes(attributes)>@content match {
+<a href="@{"".?:(href)}" class="@toClasses("govuk-back-link", classes)"@toAttributes(attributes)>@content match {
```

Comment:
since we've disabled the tests that compare output with govuk, I actually wonder if we should keep this the current way - since we know it is going to be reverted in govuk-frontend

Reply:
I think since we know we'll be going back to the "#" href, once the change comes in, we should keep it as is. I'll get this put back to include the "#".